doc/coding_standards.md

```diff
@@ -44,14 +78,39 @@ changes upstream.
 
 The best method is to isolate your feature inside of an engine in `vendor/engines`. When
 necessary, create a hook point where you need it within the open-source branch and hook
-into that with your engine. Try to avoid overriding entire views (like happens in the `c2po` engine). Then a change to the default view might need to happen in multiple places.
-
-_We have yet to find a perfect mechanism for this, but some options include:_
+into that with your engine. Try to avoid overriding entire views (like happens in
```

Comment:
I'd go with either "like in the `c2po` engine" or "as happens in the `c2po` engine"